src/config/index.ts

```diff
@@ -95,3 +95,11 @@ export const OVERSEAS_ENTITY = "overseas-entity";
 const TRANSACTION_PATH = TRANSACTION + "/:transactionId/";
 const OVERSEAS_ENTITY_PATH = OVERSEAS_ENTITY + "/:overseaEntityId/";
 export const PAYMENT_WITH_TRANSACTION_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + TRANSACTION_PATH + OVERSEAS_ENTITY_PATH + PAYMENT;
+
+// Change links
+export const PRESENTER_CHANGE_FULL_NAME = PRESENTER_URL + "#full_name";
```

Comment:
You do not need to export PRESENTER_CHANGE_FULL_NAME and CHANGE_LINKS, they are already used inside CHANGE_LINKS